examples/notebooks/bots_on_solidity.ipynb

```diff
@@ -359,19 +380,35 @@
     "fixed_math_address = sol_agents[\"agent_0\"].deploy(project.MockFixedPointMath)\n",
     "fixed_math = project.MockFixedPointMath.at(fixed_math_address)\n",
     "\n",
-    "base_ERC20.mint(int(config.target_liquidity * 10**18), sender=sol_agents[\"agent_0\"])\n",
+    "hyperdrive_math_address = sol_agents[\"agent_0\"].deploy(project.MockHyperdriveMath)\n",
+    "hyperdrive_math = project.MockHyperdriveMath.at(hyperdrive_math_address)\n",
+    "\n",
+    "base_ERC20.mint(int(config.target_liquidity * 2 * 10**18), sender=sol_agents[\"agent_0\"])\n",
     "\n",
     "initial_supply = int(config.target_liquidity * 10**18)\n",
     "initial_apr = int(config.target_fixed_apr * 10**18)\n",
     "initial_share_price = int(config.init_share_price * 10**18)\n",
     "checkpoint_duration = 86400  # seconds = 1 day\n",
-    "checkpoints_per_term = 365\n",
-    "position_duration_seconds = checkpoint_duration * checkpoints_per_term\n",
+    "checkpoints_per_term = 365#config.num_position_days # 365\n",
+    "print(f\"position_duration_days={checkpoint_duration * checkpoints_per_term / 86400}\")\n",
     "time_stretch = int(1 / simulator.market.time_stretch_constant * 10**18)\n",
+    "print(f\"{time_stretch=}\")\n",
+    "actual_time_stretch = 1 / (time_stretch / 10**18)\n",
+    "print(f\"{actual_time_stretch=}\")\n",
     "curve_fee = int(config.trade_fee_percent * 10**18)\n",
     "flat_fee = int(config.redemption_fee_percent * 10**18)\n",
-    "gov_fee = 0\n",
+    "gov_fee = int(config.governance_fee_percent * 10**18)\n",
     "\n",
+    "print(\"=== SETUP ===\")\n",
```

Comment:
exhaustive prints kept in cus they were usseful to manually inspect whether our setup conditions were identical (they were)